tests/e2e/e2e_suite_test.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	kubectl "github.com/rancher-sandbox/ele-testhelpers/kubectl"
+	"github.com/rancher-sandbox/rancheros-operator/tests/catalog"
+)
+
+var (
+	chart      string
+	externalIP string
+	magicDNS   string
+)
+
+func TestE2e(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "ros-operator e2e test Suite")
+}
+
+func waitNamespace(namespace, label string) {
```

Comment:
Maybe this function could be added in `https://github.com/rancher-sandbox/ele-testhelpers/blob/main/kubectl/kubectl.go`?

Reply:
Ok, I can do it.